sierra_adapter/Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM python:3.6-alpine
+
+LABEL maintainer "Alex Chan <a.chan@wellcome.ac.uk>"
```

Comment:
As you're not going to be the sole maintainer, probably best to make this a team rather than an individual.

Reply:
Good point.